synapse/server.py

```diff
@@ -827,3 +828,8 @@ def get_request_ratelimiter(self) -> RequestRatelimiter:
             self.config.ratelimiting.rc_message,
             self.config.ratelimiting.rc_admin_redaction,
         )
+
+    @cache_in_self
+    def get_shared_usage_metrics(self) -> SharedUsageMetrics:
```

Comment:
as a note, I'm not the biggest fan of the terminology 'shared metrics', but struggling to think of anything much better.

Slight preference for *common*, since 'shared metrics' in one sense also sounds like 'phone-home metrics', which we're trying to distinguish it from.
That said, maybe just 'usage metrics' would do anyway